icoscp/station/station.py

```diff
@@ -28,6 +28,7 @@
 from icoscp.sparql.runsparql import RunSparql
 from icoscp.sparql import sparqls
 from tqdm import tqdm
+from fmap import generate_html
```

Comment:
i got sometime an error that the module 'fmap' is not found.
I think it might be more stable using `from icoscp.station import fmap`

And since there is already an additional file (fmap) one could think to move everything from the map creation to fmap. Or the other way around, and include the HTML generation into station.py  with a private function with double underscore. At the moment the fmap file is bit disconnected.

Reply:
I changed the import statement to `import icoscp.stilt.fmap as fmap` to "match" the one in stiltMaster but if that is a problem I can still edit it as your comment.